packages/larwirecell/package.py

```diff
@@ -61,12 +61,18 @@ def patch(self):
             "find_package(Boost COMPONENTS graph date_time exception filesystem iostreams stacktrace_basic)\nfind_package(art ",
             "CMakeLists.txt",
         )
+        filter_file(
+            r"jsoncpp_lib jsonnet_lib",
+            "jsoncpp jsonnet",
+            "Modules/FindWireCell.cmake",
+        )
```

Comment:
Can you make a PR in `larwirecell` with these changes?  If not, let me know

Reply:
I will make a pull request.